platform/linux-generic/arch/default/odp/api/abi/random.h

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2021, Nokia
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+#ifndef ODP_API_ABI_RANDOM_H_
+#define ODP_API_ABI_RANDOM_H_
```

Comment:
It seems to me that these new random.h files in the default and x86 archs should not be in the abi directory as they are not part of the abi and not needed for the abi. I think they could as well be in the top level default and x86 arch directories. Or maybe I am missing something?